ProjectLighthouse.Servers.GameServer/Startup/GameServerStartup.cs

```diff
@@ -90,7 +90,7 @@ public virtual void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
             Logger.Warn
             (
-                "The serverDigestKey configuration option wasn't set, so digest headers won't be set or verified. This will also prevent LBP 1, LBP 2, and LBP Vita from working. " +
+                "The serverDigestKey configuration option wasn't set, so digest headers won't be set or verified. This will prevent LBP 1, and LBP 3 from working, and may break features in the other games (i.e Dive in). " +
```

Comment:
```suggestion
                "The serverDigestKey configuration option wasn't set, so digest headers won't be set or verified. This will prevent LBP 1 and LBP 3 from working, and may break features in the other games (i.e Dive In). " +
```